modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -3026,6 +3060,99 @@ def groupby_dtypes(
             drop=drop,
         )
 
+    @_inherit_docstrings(BaseQueryCompiler.groupby_agg)
+    def _groupby_shuffle(
+        self,
+        by,
+        agg_func,
+        axis,
+        groupby_kwargs,
+        agg_args,
+        agg_kwargs,
+        drop=False,
+        how="axis_wise",
+    ):
+        if Engine.get() == "Python":
```

Comment:
What about hdk?

Reply:
this is pandas query compiler, this method shouldn't be accessed by HDK